extensions/visualizations/models.py

```diff
@@ -76,6 +76,14 @@ class BarChart(BaseVisualization):
         'schema': {
             'type': 'unicode',
         },
+    }, {
+        'name': 'show_addressed_info',
```

Comment:
Actually, this isn't the way to do it, if you want the show_addressed_info field to exist for all visualizations. If someone adds a new visualization type, it's possible that it won't have a show_addressed_info field.

Instead I would suggest modifying the test so that they check for "existing keys + show_addressed_info", or passing in show_addressed_info separately from `options`. But treating show_addressed_info as a per-visualization customization option doesn't seem like what was originally intended...

Reply:
Done, how's that?